ctlearn/default_models/single_cnn.py

```diff
@@ -42,11 +45,15 @@ def single_cnn_model(data, model_params):
             )(network_input)
 
         engine_output = engine(network_input, params=model_params, name=backbone_name)
+        engine_output_param = engine_param(network_input_param, params=model_params, name=backbone_name)
 
         output = tf.keras.layers.GlobalAveragePooling2D(
             name=backbone_name + "_global_avgpool"
         )(engine_output)
+        output_param = tf.keras.layers.Flatten()(engine_output_param)
+        concat = tf.keras.layers.Concatenate()([output, output_param])
```

Comment:
Also try to align the naming convention here:
`concat = tf.keras.layers.Concatenate()([output_cnn, output_mlp])`

Reply:
done